### PR TITLE
[MPIR-451] Rename "Dependency Information" to "Maven Coordinates"

### PR DESCRIPTION
**Pull request by michael-o**
This closes #67

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MPIR) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MPIR-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MPIR-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Review by slawekjaranowski — APPROVED** [review]

**Comment by michael-o** [comment]
Note from me: I agree with @elharo's sentiments that this goal is incorrently named. I lines up with the other dependency goals which are about the project's deps and not about this project being a dep somewhere.

File: src/main/resources/project-info-reports_fr.properties [hunk 1]
```diff
@@ -117,8 +117,8 @@ report.dependency-convergence.stats.readyrelease.success           = Succ\u00e8s
 report.dependency-convergence.stats.snapshots                      = Nombre d'artefacts SNAPSHOT (NOS)\u202f:
 report.dependency-convergence.stats.modules                        = Nombre de sous-projets\u202f:
 report.dependency-convergence.title                                = R\u00e9action de la convergence des d\u00e9pendances
-report.dependency-info.name                                        = Informations de d\u00e9pendance
-report.dependency-info.title                                       = Informations de d\u00e9pendance
+report.dependency-info.name                                        = Coordonnu00e9es de Maven
```
Comment (elharo) [inline src/main/resources/project-info-reports_fr.properties:120]: I think you're missing a backslash before \u00e9 here and in the next line

File: src/site/apt/index.apt.vm [hunk 2]
```diff
@@ -44,8 +44,8 @@ ${project.name}
    * {{{./dependency-convergence-mojo.html}project-info-reports:dependency-convergence}} is used to generate the
    Project Dependency Convergence report for (reactor) builds.
 
-   * {{{./dependency-info-mojo.html}project-info-reports:dependency-info}} is used to generate code snippets to
-   be added to build tools.
+   * {{{./dependency-info-mojo.html}project-info-reports:dependency-info}} is used to generate code snippets
```
Comment (elharo) [inline src/site/apt/index.apt.vm:47]: is used to generate --> generates

File: src/site/apt/index.apt.vm [hunk 3]
```diff
@@ -79,6 +79,17 @@ ${project.name}
 
    * {{{./summary-mojo.html}project-info-reports:summary}} is used to generate the Project Summary report.
 
+* Upcoming Incompatibility Notice
+
+  With the next major version (4.0) mojos and output filenames will change for consistency alignment
```
Comment (elharo) [inline src/site/apt/index.apt.vm:84]: period at end

File: src/site/apt/index.apt.vm [hunk 4]
```diff
@@ -79,6 +79,17 @@ ${project.name}
...
+  To make you aware of that, see the following table for the upcoming changes:
```
Comment (elharo) [inline src/site/apt/index.apt.vm:85]: Might drop "To make you aware of that,"

File: src/site/apt/index.apt.vm (hunk 2 above)
Comment (michael-o) [reply src/site/apt/index.apt.vm:47]: This goes throughout all bulletpoints. I'd leave as is and do this in another PR.

**Review by elharo — APPROVED** [review]

**Review by slachiewicz — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #346

**Pull request merged**